### PR TITLE
Makes ninja and wizards no longer spawn with suit sensors 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Ninja now spawns with a tactical turtleneck so that they are not spawning with max suits to be pin pointed

## Why It's Good For The Game

Stops ninjas for being known as soon as they spawn by a medical officer going `Huh whos this new unknown guy in space` and then sec getting pin pointers to track them down to their location

## Changelog
:cl:
balance: The suit full of spiders also known as a ninja now is a tactical turtleneck
balance: The Wiznerds now dont have suits set to max
/:cl:

### CODE REVIEW

**Comment by KeRSedChaplain** [comment]
what if we do the same for wizards.

**Comment by Trilbyspaceclone** [comment]
@KeRSedChaplain Good idea

File: code/modules/clothing/under/color.dm [hunk 1]
```diff
@@ -196,6 +196,14 @@
 	item_state = "p_suit"
 	item_color = "lightpurple"
 
+/obj/item/clothing/under/color/lightpurple/trackless
+	name = "purple jumpsuit"
```
Comment (Ghommie) [inline code/modules/clothing/under/color.dm]: Remove, same value as parent.

File: code/modules/clothing/under/color.dm [hunk 2]
```diff
@@ -196,6 +196,14 @@
...
+	desc = "A magically colored jumpsuit. No sensors are attached!"
+	icon_state = "lightpurple"
+	item_state = "p_suit"
+	item_color = "lightpurple"
```
Comment (Ghommie) [inline code/modules/clothing/under/color.dm]: Idem.

File: code/modules/ninja/outfit.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /datum/outfit/ninja
 	name = "Space Ninja"
-	uniform = /obj/item/clothing/under/color/black
+	uniform = /obj/item/clothing/under/syndicate
```
Comment (Ghommie) [inline code/modules/ninja/outfit.dm]: This change brings a collateral melee resistance buff to space ninjas, make a no sensors black jumpsuit subtype, please.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**